src/teacher.rb

```diff
@@ -0,0 +1,16 @@
+require('./src/person')
+
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
+    super(name, age, parent_permission: parent_permission)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    true
+  end
+
+  def teach
+    'Everything in Ruby is an Object'
+  end
```

Comment:
- [ ] [Optional] It would be better if you remove this method because it's not required to make it.